p2p/node/p2p_services.go

```diff
@@ -59,6 +59,7 @@ func (p *P2PNode) requestFromPeer(peerID peer.ID, location common.Location, data
 	if err != nil {
 		return nil, err
 	}
+	log.Global.Infof("[%d] Sent request to peer: %s, %s, %s", id, stream.Conn().RemotePeer(), location.Name(), data)
```

Comment:
Why is this log Info instead of Debug?

Reply:
because we want it to be more visible right now in testing. We should reduce it back to debug later.